src/storage/src/hummock/sstable/builder.rs

```diff
@@ -704,6 +707,15 @@ impl<W: SstableWriter, F: FilterBuilder> SstableBuilder<W, F> {
                 data_len, block_meta.offset
             )
         });
+
+        if data_len as usize > self.options.capacity * 2 {
```

Comment:
Is this log necessary to keep?

Reply:
I prefer to keep this log for debugging, Therefore, we can quickly identify similar problems, such as not switching block

Reply:
IMO, that log shouldn't show up under normal circumstances.